### DISCUSSION TITLE
The folder name is strange

### DISCUSSION THREAD

**Discussion by sudosoo** [community/community#119732] [Repositories] [answered]
### Select Topic Area

Question

### Body

![스크린샷 2024-04-16 101254](https://github.com/community/community/assets/35169204/9c09accf-fb59-41aa-9b63-094a2344768f)

Only the top level folder name should be visible

However, all subfolder names are visible.

**Comment by DNin01** [comment, marked as answer]
When a folder has no files and only one subfolder inside, that happens. It's a shortcut to the last subfolder. The point is that you don't have to click through all of the folders leading up to the last folder in the chain just to get there.

![Image of subfolder path](https://github.com/community/community/assets/106490990/709cba35-1a0f-494e-8ff6-e036a089faf4)

If you need to access the other folders, you can use the breadcrumbs.

![Breadcrumbs](https://github.com/community/community/assets/106490990/4904e977-b482-410c-b5a0-385961943f1a)

**Reply by sudosoo** [reply]
tank u   :)
